app/views/works/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<% ["album", "book", "movie"].each do |type| %>
+    <table>
+        <h2><%= type.capitalize %>s</h2>
+        <tr>
+            <th>Votes</th>
+            <th>Title</th>
+            <th>Created By</th>
+            <th>Published</th>
+            <th>Upvote</th>
+        </tr>
+        <% selected_works = @works.where(category: type)%>
+        <% ordered_works = selected_works.sort_by do |work| %>
+            <% work.votes.length %>
+        <% end %>
+        <% ordered_works.reverse! %>
```

Comment:
This is an awful lot of work to be doing in a view. Could you extract this to a model method? It would probably look pretty similar to your `top_ten` method.